templates/rhpam73-authoring-ha.yaml

```diff
@@ -1269,21 +1299,174 @@ objects:
+            },
+            "name": "${APPLICATION_NAME}-amq-drainer-pod"
+          },
+          "spec": {
+            "serviceAccount": "${APPLICATION_NAME}-amq-sa",
+            "serviceAccountName": "${APPLICATION_NAME}-amq-sa",
+            "terminationGracePeriodSeconds": 5,
+            "containers": [
+              {
+                "env": [
+                  {
+                    "name": "AMQ_EXTRA_ARGS",
+                    "value": "--no-autotune"
+                  },
+                  {
+                    "name": "AMQ_USER",
+                    "value": "${APPFORMER_JMS_BROKER_USER}"
+                  },
+                  {
+                    "name": "AMQ_PASSWORD",
+                    "value": "${APPFORMER_JMS_BROKER_PASSWORD}"
+                  },
+                  {
+                    "name": "AMQ_ROLE",
+                    "value": "${AMQ_ROLE}"
+                  },
+                  {
+                    "name": "AMQ_NAME",
+                    "value": "${AMQ_NAME}"
+                  },
+                  {
+                    "name": "AMQ_TRANSPORTS",
+                    "value": "openwire"
+                  },
+                  {
+                    "name": "AMQ_QUEUES",
+                    "value": ""
+                  },
+                  {
+                    "name": "AMQ_GLOBAL_MAX_SIZE",
+                    "value": "${AMQ_GLOBAL_MAX_SIZE}"
+                  },
+                  {
+                    "name": "AMQ_ALLOW_ANONYMOUS",
+                    "value": "${AMQ_ALLOW_ANONYMOUS}"
+                  },
+                  {
+                    "name": "AMQ_DATA_DIR",
+                    "value": "/opt/amq/data"
+                  },
+                  {
+                    "name": "AMQ_DATA_DIR_LOGGING",
+                    "value": "true"
+                  },
+                  {
+                    "name": "AMQ_CLUSTERED",
+                    "value": "true"
+                  },
+                  {
+                    "name": "AMQ_REPLICAS",
+                    "value": "2"
+                  },
+                  {
+                    "name": "AMQ_CLUSTER_USER",
+                    "value": "${APPFORMER_JMS_BROKER_USER}"
+                  },
+                  {
+                    "name": "AMQ_CLUSTER_PASSWORD",
+                    "value": "${APPFORMER_JMS_BROKER_PASSWORD}"
+                  },
+                  {
+                    "name": "POD_NAMESPACE",
+                    "valueFrom": {
+                      "fieldRef": {
+                        "fieldPath": "metadata.namespace"
+                      }
+                    }
+                  },
+                  {
+                    "name": "OPENSHIFT_DNS_PING_SERVICE_PORT",
+                    "value": "8888"
+                  }
+                ],
+                "image": "registry.access.redhat.com/amq-broker-7/amq-broker-72-openshift:1.1",
```

Comment:
Why doesn't this use the ${IMAGE} (or as suggested, ${AMQ_IMAGE}) template parameter defined above?